wen-restart/src/last_voted_fork_slots_aggregate.rs

```diff
@@ -28,13 +31,14 @@ pub(crate) struct LastVotedForkSlotsEpochInfo {
     // Total stake of active peers in this epoch, no matter they voted for a slot
     // in this epoch or not.
     pub actively_voting_stake: u64,
-    // Total stake of active peers which has voted for a slot in this epoch.
+    // Total stake of active peers which has voted for any slot in this epoch.
+    // Note that if last_vote slot belongs to epoch n, then this validator should
+    // have voted for at least one slot in epoch n - 1, so it should be counted
+    // in earlier epoch as well.
```

Comment:
nit: 

`Note that if last_vote slot belongs to epoch n, then this validator should then this validator should have voted for at least one slot in epoch n - 1` isn't necesarily true, the validator could have not voted at all in the previous epoch. I think it's more accurate to say 

`Note that if last_vote slot belongs to epoch n, then this validator has also implicitly voted on epoch n - 1`